.circleci/job/shared/artifact.sh

```
@@ -8,6 +8,10 @@ cp -r coverage artifact/
 cp -r recordings artifact/
 cp -r swagger artifact/
 
+# log
+cp -r *.log artifact/
```

Shellcheck (0.10.0)

[info] Use ./*glob* or -- *glob* so names with dashes won't become options.

(SC2035)